### ISSUE TITLE
Pipeline tool has crashed: Gtk.Widget.DrawnSignalCallback(IntPtr inst, IntPtr arg0, IntPtr gch)

### ISSUE THREAD

**Issue by KRC2000** [MonoGame/MonoGame#7689]
MGCB Tool is not working, error screen shown.

Here are the commands that was executed to install the tool:
![image](https://user-images.githubusercontent.com/47914319/163227807-30e65533-70b7-40ed-b322-2cb40be9c7a3.png)

After, when trying to open .mgcb file of my project, this screen is shown:
![image](https://user-images.githubusercontent.com/47914319/163228006-d4fc3b6c-07c2-4518-83d8-9c122ee34e8c.png)
that says:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Gtk.Widget.DrawnSignalCallback(IntPtr inst, IntPtr arg0, IntPtr gch)
```

Also, now that I am writing this, I remember that I had similar issue on my Linux Mint while ago.
Right now I will try to use CLI version, but it would be really cool to get ui version working. 
Thanks for your work and cheers!

#### Version of mgcb-editor:
- Version 3.8.0.1641

#### What operating system are you using:
- Pop_OS by System76 21.10

#### What MonoGame platform are you using:

- DesktopGL

**Comment by vsjoqvist** [comment]
this occurs on all linux distros, there is an aur version for the arch, not sure if there is one for popos!? i'd take a look in the pop shop

**Comment by Beyley** [comment]
duplicate issue

**Comment by KRC2000** [comment]
> duplicate issue

Can you give a link?

**Comment by vsjoqvist** [comment]
https://github.com/MonoGame/MonoGame/issues/7491

**Comment by mrhelmut** [comment]
Assuming it be fixed for the upcoming 3.8.1 release. Feel free to reopen if it persists on 3.8.1.
